2-mandatory/2-financial-times.js

```diff
@@ -5,33 +5,69 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
```

Comment:
The implementation here looks good 👍 
I would suggest giving the `ARTICLE_TITLES` a different name. It may be confusing because there's another variable with the same name on this page.
Can you think of a different name for it?